src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -21,6 +21,12 @@
     :items="props.items ? transformToTableData(props.items) : undefined"
     :error="props.error"
     @change="emit('change', $event)"
+    @row:click="router.push({
+      name: props.gateways ? 'gateway-detail-view' : 'data-plane-detail-view',
+      params: {
+        dataPlane: $event.name,
+      },
+    })"
```

Comment:
**Comment**: Most of these will change once we add summary trays to the detail views. For now, we keep the "click row to go to detail view" behavior.